posts/2022/01/ipython-8.0-lessons-learned-maintaining-software.md

```diff
@@ -0,0 +1,303 @@
+<!--
+.. title: IPython 8.0, Lessons learned maitaining software
+.. slug: ipython-8.0-lessons-learned-maintaining-software
+.. date: 2022-01-10 02:51:58 UTC-05:00
+.. author: Matthias Bussonnier
+.. tags: IPython, Software, Open-Source, best-practice
+.. category:
+.. link:
+.. description:
+.. type: text
+.. previewimage: 
+-->
+
+This is a companion post from the [Official release of IPython 8.0], that describe what we leaned with this large new
+major IPython release. We hope it will help you apply best practices, and have an easier time maintaining your projects,
+or helping other. Trust me your future self will thank you. 
```

Comment:
"Trust me your future self will thank you" feels like a promise into the future and runs counter to your statement about deprecations at the end:

> I thus prefer to stay factual, and I don't claim to predict the future.

My preference is to remove it and to combine this paragraph with "We'll focus on many patterns".

Reply:
+1